lua/weapons/remotecontroller.lua

```diff
@@ -111,3 +113,9 @@ function SWEP:Think()
 		end
 	end
 end
+
+hook.Add("PlayerNoClip", "wire_remotecontroller_antinoclip", function(ply, cmd)
+	if not ply.RCOverride then return end
+	return false
```

Comment:
Could you just change this to `if ply.RCOverride then return false end` and get rid of the following line? I don't see any other issues.